.storybook/preview.tsx

```diff
@@ -11,6 +11,16 @@ const preview: Preview = {
 				date: /Date$/i,
 			},
 		},
+		a11y: {
+			config: {
+				rules: [
+					{
+						id: 'color-contrast',
+						enabled: false,
+					},
+				],
+			},
+		},
```

Comment:
Any reason why we have disabled this?

Reply:
@vrundakansara, we replaced some colors with new ones Akbar provided for the Color contrast accessibility issue, but it still was showing the color contrast ratio violation.

Akbar suggested ignoring the warning and then we disabled the rule.